packages/cht/README.md

```diff
@@ -0,0 +1,31 @@
+# language-cht <img src='./assets/square.png' width="30" height="30"/>
+
+An OpenFn **_adaptor_** for building integration jobs for use with the cht API.
+
+## Documentation
+
+View the [docs site](https://docs.openfn.org/adaptors/packages/cht-docs)
+for full technical documentation.
+
+### Configuration
+
+View the
+[configuration-schema](https://docs.openfn.org/adaptors/packages/cht-configuration-schema/)
+for required and optional `configuration` properties.
+
+### A example expression using the `create` function
+
+```js
+post('patient', { name: 'Bukayo' });
```

Comment:
Can we make this a more realisic/relevant example? The title will need updating too